test/timeout.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+. $(dirname "${BASH_SOURCE[0]}")/common.sh
+
+PORT=$(($$ + 1024)) # PID + 1024
```

Comment:
Good, old std.socket can get you a free port too:

```
rdmd --eval='import std.socket; auto s = new Socket(AddressFamily.INET, SocketType.STREAM); scope(exit) s.close; s.bind(new InternetAddress(0));s.localAddress.toPortString.writeln;'
```

Reply:
It's fine there won't be colliding PIDs.